dev/integration_tests/android_semantics_testing/test_driver/main_test.dart

```diff
@@ -1,702 +0,0 @@
-import 'package:android_semantics_testing/test_constants.dart';
-import 'package:flutter_driver/flutter_driver.dart';
-import 'package:path/path.dart' as path;
-import 'package:pub_semver/pub_semver.dart';
-import 'package:test/test.dart' hide isInstanceOf;
-
-// The accessibility focus actions are added when a semantics node receives or
-// lose accessibility focus. This test ignores these actions since it is hard to
-// predict which node has the accessibility focus after a screen changes.
-const List<AndroidSemanticsAction> ignoredAccessibilityFocusActions = <AndroidSemanticsAction>[
-  AndroidSemanticsAction.accessibilityFocus,
-  AndroidSemanticsAction.clearAccessibilityFocus,
-];
-
-String adbPath() {
-  final String? androidHome = io.Platform.environment['ANDROID_HOME'] ?? io.Platform.environment['ANDROID_SDK_ROOT'];
-  if (androidHome == null) {
-    return 'adb';
-  } else {
-    return path.join(androidHome, 'platform-tools', 'adb');
-  }
-}
-
-void main() {
-  group('AccessibilityBridge', () {
-    late FlutterDriver driver;
-    Future<AndroidSemanticsNode> getSemantics(SerializableFinder finder) async {
-      final int id = await driver.getSemanticsId(finder);
-      final String data = await driver.requestData('getSemanticsNode#$id');
-      return AndroidSemanticsNode.deserialize(data);
-    }
-
-    // The version of TalkBack running on the device.
-    Version? talkbackVersion;
-
-    Future<Version> getTalkbackVersion() async {
-      final io.ProcessResult result = await io.Process.run(adbPath(), const <String>[
-        'shell',
-        'dumpsys',
-        'package',
-        'com.google.android.marvin.talkback',
-      ]);
-      if (result.exitCode != 0) {
-        throw Exception('Failed to get TalkBack version: ${result.stdout as String}\n${result.stderr as String}');
-      }
-      final List<String> lines = (result.stdout as String).split('\n');
-      String? version;
-      for (final String line in lines) {
-        if (line.contains('versionName')) {
-          version = line.replaceAll(RegExp(r'\s*versionName='), '');
-          break;
-        }
-      }
-      if (version == null) {
-        throw Exception('Unable to determine TalkBack version.');
-      }
-
-      // Android doesn't quite use semver, so convert the version string to semver form.
-      final RegExp startVersion = RegExp(r'(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)(\.(?<build>\d+))?');
-      final RegExpMatch? match = startVersion.firstMatch(version);
-      if (match == null) {
-        return Version(0, 0, 0);
-      }
-      return Version(
-        int.parse(match.namedGroup('major')!),
-        int.parse(match.namedGroup('minor')!),
-        int.parse(match.namedGroup('patch')!),
-        build: match.namedGroup('build'),
-      );
-    }
-
-    setUpAll(() async {
-      driver = await FlutterDriver.connect();
-      talkbackVersion ??= await getTalkbackVersion();
-      print('TalkBack version is $talkbackVersion');
```

Comment:
Looks like this no longer prints in the new test? Do we want to keep this? Could be useful for debugging...

Reply:
added back to enableTalkBack